src/playbooks/foreman-proxy/metadata.obsah.yaml

```diff
@@ -0,0 +1,8 @@
+---
+help: |
+  Install Foreman Proxy
+
+include:
+  - _certificate_source
+  - _database_mode
+  - _database_connection
```

Comment:
The proxy itself doesn't have a database, so why are these needed?

Reply:
I think I'd drop it for now